src/collection-view.js

```diff
@@ -35,67 +36,43 @@ var CollectionView = Backbone.View.extend({
 
     this._setOptions(options);
 
-    MonitorDOMRefresh(this);
-
-    this.on({
-      'before:show':   this._onBeforeShowCalled,
-      'show':          this._onShowCalled,
-      'before:attach': this._onBeforeAttachCalled,
-      'attach':        this._onAttachCalled
-    });
+    MonitorViewEvents(this);
 
     this._initBehaviors();
     this.once('render', this._initialEvents);
     this._initChildViewStorage();
-    this.initRenderBuffer();
+    this._resetRenderBuffer();
 
     Backbone.View.prototype.constructor.call(this, this.options);
 
     this.delegateEntityEvents();
   },
 
-  // Instead of inserting elements one by one into the page,
-  // it's much more performant to insert elements into a document
-  // fragment and then insert that document fragment into the page
-  initRenderBuffer: function() {
+  // Instead of inserting elements one by one into the page, it's much more performant to insert
+  // elements into a document fragment and then insert that document fragment into the page
+  _resetRenderBuffer: function() {
```

Comment:
So this is a better name, although I'm now wondering why we don't just change line 58 to `this._bufferedChildren = [];`

I don't think it needs to be initialized, and everything that needs it is called after `_startBuffering`